conslogging/conslogging_test.go

```diff
@@ -54,7 +54,7 @@ func Test_prettyPrefix(t *testing.T) {
 			name:          "simple target with path",
 			prefixPadding: DefaultPadding,
 			prefix:        "github.com/earthly/earthly+run",
-			expected:      "github.com/earthly/earthly+run",
```

Comment:
this is a regression that was fixed, before earthly `v0.7.0` we used to shorten these urls, since `v0.7.0`
we only shorten addresses such as `github.com/earthly/earthly:80524f0d82a353b3444e83f056207e15f4d5447c+hello-world`